grafana-plugin/src/state/plugin.ts

```diff
@@ -31,6 +31,11 @@ export async function startPluginSync() {
   return await makeRequest('/plugin/sync', { method: 'POST' });
 }
 
+
+export const SYNC_STATUS_RETRY_LIMIT = 10;
+
+export const syncStatusDelay = retryCount => new Promise(resolve => setTimeout(resolve, 10 * 2 ** retryCount));
```

Comment:
Looks good but I wonder why do we need to calculate it based on `retryCount` and instead not relying on a default number? Previously we had `2000ms`. It feels slightly odd to have a sequence of timeouts - `20, 40, 80, 160...` and so on, add to that that timeouts lower than `200-300ms` aren't really noticeable.